src/test/java/gov/usgs/wma/waterdata/DBTestConfig.java

```diff
@@ -0,0 +1,90 @@
+package gov.usgs.wma.waterdata;
+
+import java.sql.SQLException;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+
+import com.github.springtestdbunit.bean.DatabaseConfigBean;
+import com.github.springtestdbunit.bean.DatabaseDataSourceConnectionFactoryBean;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@TestConfiguration
+public class DBTestConfig {
+
+	// Transform DB connection
+	@Value("${TRANSFORM_SCHEMA_NAME}")
+	private String transformSchemaName;
+
+	@Bean
+	@Primary
+	@ConfigurationProperties(prefix="spring.datasource-transform")
+	public DataSourceProperties dataSourcePropertiesTransform() {
+		return new DataSourceProperties();
+	}
+
+	@Bean
+	@Primary
+	@ConfigurationProperties(prefix="spring.datasource-transform")
```

Comment:
The DBConfig doesn't have the @ConfigurationProperties on the DataSource beans. If it's really not needed we should remove it from here and dataSourceObservation(). (Or visa-versa should be added in DBConfig)

Reply:
Went with your latter suggestion and added the annotations to the DataSource beans in the DBConfig.